debian/mariadb-server.README.Debian

```diff
@@ -84,14 +84,14 @@ MariaDB in Debian is secure by default, because:
 - There is no root account with password anymore. The system admin needs to
   create one themselves if they need it. With no password, all issues related
   to password management and password leaking are gone. Sysadmins can access
-  the database without a password simply by running 'sudo mysql' thanks to
+  the database without a password simply by running 'sudo mariadb' thanks to
   socket based authentication, which detects the system root user and allows
-  them to use the mysqld console as the mysql root user. For details see
+  them to use the mariadb console as the MariaDB root user. For details see
```

Comment:
How do we choose to use MariaDB or mariadb?

Reply:
"mariadb" is the console application formerly known as "mysql". "MariaDB root user" is referring to the user `root` inside the MariaDB server.